filter_plugins/filter_snake_case.py

```diff
@@ -0,0 +1,10 @@
+# copied from https://github.com/infrawatch/tripleo-qdr-ansible-role/commit/4897cb6a09fd83c12b8d7a578c845471afe68dbc
+import re
+
+def filter_snake_case(camel_case_string):
+  name = re.sub(r'(.)([A-Z][a-z]+)', r'\1_\2', camel_case_string)
```

Comment:
There is actually a bug here, this should be `camel_case_string=...` See https://review.opendev.org/c/openstack/tripleo-ansible/+/772920/2/tripleo_ansible/roles/tripleo_metrics_qdr/filter_plugins/filter_snake_case.py

But also see https://review.opendev.org/c/openstack/tripleo-ansible/+/772920/1/tripleo_ansible/roles/tripleo_metrics_qdr/filter_plugins/filter_snake_case.py "Role filters should go in the (tripleo-ansible) filters directory